Marlin/ultralcd.cpp

```diff
@@ -830,14 +831,13 @@ static void _lcd_move(const char* name, AxisEnum axis, int min, int max) {
     if (min_software_endstops) NOLESS(current_position[axis], min);
     if (max_software_endstops) NOMORE(current_position[axis], max);
     encoderPosition = 0;
-    if (movesplanned() <= 3)
-      line_to_current(axis);
```

Comment:
Here `current_position` has already been set, but if the planner is full the move is just dropped.